docs/deploy.rst

```diff
@@ -64,11 +64,11 @@ Environments
 Will this run in Heroku? Probably, but you'll need to do some Heroku footwork to
 set it up.
 
-Will this run on AWS? Yes--that's what we do.
+Will this run on GCP? Yes--that's what we do.
```

Comment:
I should do a pass on removing as much of the Antenna docs as possible. There's a lot of stuff in here to help other companies/people run Antenna and we don't support that.